libdebug/architectures/aarch64/aarch64_breakpoint_validator.py

```diff
@@ -0,0 +1,24 @@
+#
+# This file is part of libdebug Python library (https://github.com/libdebug/libdebug).
+# Copyright (c) 2024 Roberto Alessandro Bertolini. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for details.
+#
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from libdebug.data.breakpoint import Breakpoint
+
+
+def validate_breakpoint_aarch64(bp: Breakpoint) -> None:
+    """Validate a hardware breakpoint for the AARCH64 architecture."""
+    if bp.condition not in ["r", "w", "rw", "x"]:
```

Comment:
Didn't we establish that "w" is not a valid condition? BTW, this is likely a leftover in other functions and should be corrected elsewhere as well.

Reply:
This is another thing where, unfortunately, different architectures have different behaviors. Even if we strive for consistency, I don't think we should restrict the conditions if the hardware has the support for them.

Reply:
This asymmetry is now documented, so I consider my skill issue resolved